worldedit-core/src/main/java/com/sk89q/worldedit/internal/expression/ExpressionHelper.java

```diff
@@ -49,14 +55,18 @@ public static EvaluationException evalException(ParserRuleContext ctx, String me
     }
 
     public static EvaluationException evalException(Token token, String message) {
+        return evalException(getErrorPosition(token), message);
+    }
+
+    public static EvaluationException evalException(int positionInLine, String message) {
         return new EvaluationException(
-                getErrorPosition(token),
-                message
+            positionInLine,
+            message
         );
     }
 
-    public static void checkIterations(int iterations, ParserRuleContext ctx) {
-        check(iterations <= 256, ctx, "Loop exceeded 256 iterations");
+    public static void checkIterations(int iterations, int positionInLine) {
+        check(iterations <= 256, positionInLine, "Loop exceeded 256 iterations");
```

Comment:
Is this definitely keeping the same behaviour?

Reply:
Seems like it, change from upstream and references to the method are updated as well (called with `.positionInLine()`)